File: CahBackend/Models/Deck/Cards.cs
using CahBackend.Models.Deck;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CahBackend.Models.Deck
{
    public class Cards
    {
        public List<BlackCard> BlackCards { get; set; }
        public List<WhiteCard> WhiteCards { get; set; }
        public HistoryCards HistoryCards { get; set; }
    }
}

File: CahFrontend/src/app/game/models/Cards.ts
export interface HistoryCards {
  blackCards: { [key: number]: BlackCard };
  whiteCards: { [key: number]: WhiteCard };
}

export interface BlackCard {
  pick: number;
  text: string;
}

export interface WhiteCard {
  text: string;
}

export interface Cards {
  blackCards: BlackCard[];
  whiteCards: WhiteCard[];
  historyCards: HistoryCards;
}
